et_replay/lib/comm/commsTraceParser.py

```diff
@@ -2,14 +2,12 @@
 from __future__ import annotations
 
 import json
-
 from typing import List, Tuple
 
-from et_replay.lib.comm import comms_utils
-from et_replay.lib.comm.comms_utils import commsArgs
-from et_replay.lib.comm.pytorch_backend_utils import supportedP2pOps
-
-from et_replay.lib.execution_trace import ExecutionTrace
+from .. import ExecutionTrace
+from . import comms_utils
+from .comms_utils import commsArgs
+from .pytorch_backend_utils import supportedP2pOps
```

Comment:
can we keep the imports absolute? our absolute import strings are not really that long for us to justify relative imports. they also make code refactoring tricky. absolute imports are also more readable and pep8 friendly.